src/api/dependency-graph/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "dependency-graph",
```

Comment:
Can we rename this?  It's not a `graph` so much as a service for inspecting and discovering our dependencies.  What about calling it `dependency-discovery` and changing the directory name too?